mobile/components/Text/index.js

```diff
@@ -45,8 +45,8 @@ const monospaceFont = Platform.OS === 'android' ? 'monospace' : 'Menlo';
 const Text: ComponentType<Props> = styled.Text`
   font-weight: ${props =>
     props.weight ? `${getWeightFromType(props.weight)}` : '400'};
-  font-size: ${props => getTypeConfigFromType(props.type).size}px;
-  line-height: ${props => getTypeConfigFromType(props.type).leading}px;
+  font-size: ${props => getTypeConfigFromType(props.type).size};
+  line-height: ${props => getTypeConfigFromType(props.type).leading};
```

Comment:
As you can see, the biggest different between vanilla RN and react-primitives (apart from the minimal API) is that styled-components doesn't do any fancy "make it look like web CSS" tricks—instead it just takes the string and turns it into an object, so `px` is not required anymore.